packages/react/src/components/builder-block.component.tsx

```diff
@@ -1,5 +1,6 @@
 /** @jsx jsx */
-
+/* ts-ignore */
+// @ts-ignore
```

Comment:
Is this still needed? Ideally we'd not persist the ts-ignore, completely removes type-checking

Reply:
done, removed.